modules/ROOT/pages/vpn-maintenance.adoc

```diff
@@ -0,0 +1,89 @@
+= Anypoint VPN Maintenance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+== Avoiding or reducing downtime during an update
+
+Mule applications relying upon Anypoint VPN connectivity may be impacted during the maintenance period. However, the impact on the Anypoint VPN varies depending on the VPN configuration:
+
+* An Anypoint VPN with at least one pair of tunnels configured with BGP routing and DPD (Dead Peer Detection) to reset the tunnels when triggered incurs minimal downtime. The switchover takes a few seconds. See https://help.mulesoft.com/s/article/DPD-Dead-Peer-Detection-and-Anypoint-VPN-Virtual-Private-Networking[this article^] for an explanation of DPD and VPN.
+* An Anypoint VPN tunnel pair configured with Static routing incurs some downtime before reconnection. This takes longer if you do not have an automated failover mechanism to detect failed tunnels.
+* For an Anypoint VPN configured without BGP routing, a failover mechanism, or if only one tunnel is configured, the outage is up to 60 minutes long as the update occurs and the original tunnel comes back up.
+
+== Anypoint VPN update
+
+As part of required regular maintenance, and with the aim of maintaining good security hygiene, MuleSoft upgrades the infrastructure and configuration to power your VPN connectivity.
```

Comment:
As part of required > As part of **the** required
with the aim of maintaining > to maintain **a** good...